openprescribing/pipeline/management/commands/run_pipeline_e2e_tests.py

```diff
@@ -68,7 +68,6 @@ def run_end_to_end():
     client.create_table("ccgs", schemas.CCG_SCHEMA)
     client.create_table("stps", schemas.STP_SCHEMA)
     client.create_table("regional_teams", schemas.REGIONAL_TEAM_SCHEMA)
-    client.create_table("ppu_savings", schemas.PPU_SAVING_SCHEMA)
```

Comment:
Just to sanity check that none of our colleagues mind this disappearing...?

Reply:
As discussed at the DL day, Rich is the only person who used this and he'd be happy with just a CSV download option